tracer/src/Datadog.Trace/Ci/Coverage/DefaultCoverageEventHandler.cs

```diff
@@ -20,6 +20,11 @@ protected override object OnSessionFinished(CoverageInstruction[] coverageInstru
 
             var groupByFiles = coverageInstructions.GroupBy(i => i.FilePath).ToList();
             var coveragePayload = new CoveragePayload();
+            if (Tracer.Instance?.ActiveScope?.Span is { } span)
```

Comment:
In every case where `Ci.Coverage.CoverageReporter.Handler.EndSession()` is called, there's an active span. Rather than having a static `Tracer.Instance` access here, I'd rather we added a parameter to `EndSession(Span? span)`. That might reduce testing issues later?

EDIT: just seen the addition of `StopCoverage(Span span)` in `Testing.Common`. Given that, are the changes here even necessary? 🤔

Reply:
No, you're right. I will remove this.

Reply:
done in [4ce82ef](https://github.com/DataDog/dd-trace-dotnet/pull/2969/commits/4ce82ef50fb5b472f964b1231b96bcc6eb46bf8a)